addons/account/views/account_menuitem.xml

```diff
@@ -11,7 +11,6 @@
             <menuitem id="menu_action_move_out_invoice_type" action="action_move_out_invoice_type" sequence="1"/>
             <menuitem id="menu_action_move_out_refund_type" action="action_move_out_refund_type" sequence="2"/>
             <menuitem id="menu_action_move_out_receipt_type" action="action_move_out_receipt_type" groups="account.group_sale_receipts" sequence="3"/>
-            <menuitem id="menu_action_open_sale_payment_items" name="Amounts to Settle" action="action_open_sale_payment_items" sequence="10"/>
```

Comment:
AFAIK, `action_open_sale_payment_items` is never accessible anymore!?
Isn't that removing an entire feature?

Reply:
The Sales part is still reachable trhough the Aged Receivable. 17.0 doesn't have a dashboard access for amounts to settle linked to invoices but it comes with 17.2. So the feature isn't really lost, it just comes in a better version later. We kinda cancel it in 17.0 though, yes.

Reply:
I don't see the action being used anywhere in master, can you show me where it is done?
And should we do this in stable actually? I don't think so, people don"t expect menus to disappear after an upgrade...

Reply:
Aged Reports: 
Available from 17.0 via the method `aged_partner_balance_audit` which uses the action `account.action_open_payment_items`. Can be found in `enterprise/account_reports/models/account_aged_partner_balance.py:376`

Dashboard:
Available from 17.0 for Bills with the action `action_open_payment_items` and will be available for Invoices as soon as [this](https://github.com/odoo/odoo/pull/158014) is merged.